### ISSUE TITLE
Report Archive View makes it hard to find old report

### ISSUE THREAD

**Issue by er4z0r** [GhostManager/Ghostwriter#564]
**Describe the bug**
I've given the archiving feature a spin and my general impression is that it could use some love. The archives view lists only the client and a link to a zip archive which isn't optimal should you ever be looking for an archived report.

**To Reproduce**
1. Create two or more projects for the same client. Give each of them a report and set the end-date to  be 91 days in the past (so it will be archived)
2. Run the `ghostwriter.reporting.tasks.archive_projects` scheduled task with no args
3. Browse to Archived Reports
4. See list containing only client name and icon/link to report archive. No way to tell which report is which.

**Expected Behavior**
A list containing both client name, project name, report name and a link to the report.

**Screenshots**

**Server Specs:**
 - Ghostwriter 4.3.9

**Comment by chrismaddalena** [comment]
Yes, the archive feature needs some significant updates. It's on the radar for upcoming work. Adding a report name to the table should be easy enough for addressing this particular UX issue.

**Comment by chrismaddalena** [comment]
For now, I've added a column to the table with the project name to help identify the reports. This does not help identify archived reports if a project has multiple, but it's a start. When we revisit archiving, we'll probably need to expand that model to include a report name. I've tracked that for the rework in early 2025. This initial change will be in the next release that I'll cut in a bit.
